buddyboss/buddyboss-platform
# PROD-717 Fix - Edit Profile type -> Search related setting should be available only if Network search is enabled

### All Submissions:


Fixes #3067 

### How to test the changes in this Pull Request:

1. Disable Network Search at BuddyBoss -> Settings -> Search
2. Now, go to Edit Profile Type page
3. See Network Search related setting under Members Directory is still available
4. Now, It is only available only if network search enabled

### Proof Screenshots or Video
Search component is disabled: https://ibb.co/whtMkLk
When the Search component is enabled search-related setting is shown: https://ibb.co/qJ6KHcP but after disabling it, it is not shown: https://ibb.co/r2GXK67

### Other information:
